libraries/AP_Mount/AP_Mount_Alexmos.cpp

```diff
@@ -39,9 +40,19 @@ void AP_Mount_Alexmos::update()
 
         // point to the angles given by a mavlink message
         case MAV_MOUNT_MODE_MAVLINK_TARGETING:
-            // do nothing because earth-frame angle targets (i.e. _angle_ef_target_rad) should have already been set by a MOUNT_CONTROL message from GCS
+        {
+#define GIMBAL_YAW_SCALE (1.0f/20.0f)
+            AP_Mount *mount = AP::mount();
+            if (mount->mount_yaw_follow_mode == AP_Mount::gimbal_yaw_follows_vehicle)
```

Comment:
Check pointer (even though I think it should be fine in this class)

Reply:
Done